doc/user/content/sql/create-materialized-view.md

```diff
@@ -1,78 +1,92 @@
 ---
 title: "CREATE MATERIALIZED VIEW"
-description: "`CREATE MATERIALIZED VIEW` creates a materialized view, which Materialize will incrementally maintain as updates occur to the underlying data."
+description: "`CREATE MATERIALIZED VIEW` defines a view that is persisted in storage and incrementally updated as new data arrives."
 menu:
   main:
     parent: 'commands'
 ---
 
-`CREATE MATERIALIZED VIEW` creates a materialized view, which lets you retrieve
-incrementally updated results of a `SELECT` query very quickly. Despite the
-simplicity of creating a materialized view, it's Materialize's most powerful
-feature.
+`CREATE MATERIALIZED VIEW` defines a view that is persisted in storage and
+incrementally updated as new data arrives.
 
-## Conceptual framework
-
-`CREATE MATERIALIZED VIEW` computes and maintains the results of a `SELECT`
-query in memory. For more information, see [Key Concepts: Materialized views](/overview/key-concepts/#materialized-views).
+A materialized view specifies a [cluster](/overview/key-concepts/#clusters) that
+is tasked with maintaining its results up-to-date, but **can be referenced in
```

Comment:
"maintaining its results up-to-date" -> "maintaining its results" or "keeping its results up-to-date"